net/rhp/contracts.go

```diff
@@ -117,29 +112,29 @@ func ValidateContractRenewal(existing, renewal types.FileContract, currentHeight
 	case renewal.RevisionNumber != 0:
 		return errors.New("revision number must be zero")
 	case renewal.Filesize != existing.Filesize:
-		return errors.New("file size must not change")
+		return errors.New("filesize must not change")
 	case renewal.FileMerkleRoot != existing.FileMerkleRoot:
-		return errors.New("file merkle root must not change")
+		return errors.New("file Merkle root must not change")
 	case renewal.WindowEnd < existing.WindowEnd:
-		return errors.New("renewal window end must be after existing window end")
+		return errors.New("renewal window end must be after current window end")
```

Comment:
If we want to allow funding only renewals should this be `<=`?

Reply:
I think it's correct as written -- `renewal.WindowEnd == existing.WindowEnd` will *not* raise an error. The error message is a bit misleading, though